src/main/java/eu/dissco/nusearch/service/RabbitMQService.java

```diff
@@ -6,36 +6,54 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import eu.dissco.nusearch.domain.DigitalSpecimenEvent;
+import eu.dissco.nusearch.property.RabbitMQProperties;
 import java.util.List;
 import java.util.Objects;
 import lombok.AllArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
 import org.springframework.context.annotation.Profile;
-import org.springframework.kafka.annotation.KafkaListener;
 import org.springframework.messaging.handler.annotation.Payload;
 import org.springframework.stereotype.Service;
 
-@Service
+
 @Slf4j
+@Service
 @Profile({STANDALONE, S3_RESOLVER})
 @AllArgsConstructor
-public class KafkaConsumerService {
+public class RabbitMQService {
 
   private final ObjectMapper mapper;
+  private final RabbitTemplate rabbitTemplate;
   private final DigitalSpecimenMatchingService service;
+  private final RabbitMQProperties rabbitMQProperties;
 
-
-  @KafkaListener(topics = "${kafka.consumer.topic}")
+  @RabbitListener(queues = "#{rabbitMQProperties.queueName}", containerFactory = "consumerBatchContainerFactory")
```

Comment:
change to `${rabbitMQProperties.queueName}`

Reply:
This works slightly different, this way it takes the queueName from the rabbitMqProperties for which we have set a default there.
https://docs.spring.io/spring-amqp/reference/amqp/message-converters.html

Reply:
Isn't that what `${kafka.consumer.topic}` was doing? I'm just looking at the "#" instead of "$"